### ISSUE TITLE
2.1.2.2 Registration is running.

### ISSUE THREAD

**Issue by roywill** [ietf-wg-scitt/draft-ietf-scitt-scrapi#22]
The key returned in the example is the identifier of the payload or the signed statement as they are the same HASH.  If it is the payload, this precludes simultaneously registering two signed statements simultaneously.  I suspect you really want to supply a transaction token not reference anything about the signed statement nor the payload.

**Comment by JAG-UK** [comment]
Agreed that name scheme doesn't work. There are a couple of different problems here:
* the collision risk identified remains a problem for durable identifiers for Receipts or registered Signed Statements. We need to tackle that separately
* for the identifier returned in a 'registration is running' case, we need to return an ID for the _operation_ against which the client polls. Attention should be given to properties of this ID such as surviving pod crashes.

We implemented this for the 120 hackathon, I'll put up a R and the group can comment.

**Comment by JAG-UK** [comment]
In my PR for Issue #23 I changed 'receipt' to 'identifier' which opens the door to fixing this. Probably needs more explanation but don't want to change this until we've had the discussion on PR #27
